Sources/Kevin/Accounts/LinkingSession/KevinAccountLinkingSessionConfiguration.swift

```diff
@@ -131,6 +131,7 @@ public class KevinAccountLinkingSessionConfiguration {
         ///
         /// - Parameters:
         ///   - type: desired linking type.
+        @available(*, deprecated, message: "This method will be removed in the future versions of the SDK. You can safely remove it from you configuration.")
```

Comment:
```suggestion
        @available(*, deprecated, message: "This method will be removed in the future versions of the SDK. You can safely remove it from your configuration.")
```

Reply:
Why do we even leave this parameter? As far as I can see `linkingType` enum was reduced to 1 value - `bank`. 

If this parameter was left with a deprecation message in order for client app not breaking upon update - they would break anyway (I assume) because enum is missing values. And in general - enum with just one value seems a bit obsolete as all related code will just have 1 scenario 

P.S. same applies to `setPaymentType`

Reply:
The case is that none of the merchants uses SDK for card payments but they definitely use methods `setPaymentType(.bank)` and `setLinkingType(.bank)`. We agreed to not remove this methods and leave single value enums to avoid breaking changes for merchants and just mark it as deprecated.

Reply:
Understandable 👍